src/actions/handleVisibilitybutton.js

```diff
@@ -0,0 +1,8 @@
+export function handleVisibilitybutton(data){
```

Comment:
Change function name to checkSectionVisibility or something that points to function served by the code.